modules/client/test/integration/client/decoding.test.ts

```diff
@@ -199,6 +199,33 @@ describe("Client", () => {
         fail("TokenType should be ERC721");
       }
     });
+      it("Should decode an encoded withdraw action of an erc1155 token", async () => {
+        const data =
+          "0xf242432a00000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000001400000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
in general it would be a good practice to explain what is incoded in this data, or what are the params that r encoded in a comment.
i only put this here but it applies to all places